lib/dataSource-core.js

```diff
@@ -234,12 +244,11 @@ kettle.dataSource.defaultiseOptions = function (componentOptions, options, direc
 
 /** Operate the core "transforming promise workflow" of a dataSource's `get` method. Gets the "initial payload" from the dataSource's `getImpl` method
  * and then pushes it through the transform chain to arrive at the final payload.
- * @param that {Component} The dataSource itself
- * @param directModel {Object} The direct model expressing the "coordinates" of the model to be fetched
- * @param options {Object} A structure of options configuring the action of this get request - many of these will be specific to the particular concrete DataSource
+ * @param {Component} that - The dataSource itself
+ * @param {Object} directModel - The direct model expressing the "coordinates" of the model to be fetched
+ * @param {Object} options - A structure of options configuring the action of this get request - many of these will be specific to the particular concrete DataSource
```

Comment:
Given that options is optional in the function you're calling, can it also be flagged as optional here?